### DISCUSSION TITLE
ngrok?

### DISCUSSION THREAD

**Discussion by jaxxtheone** [Guilouz/Creality-Helper-Script-Wiki#616] [Features Request]
Is it possible to have it installed? What´s the exact name of the Linux flavor we have on K1x? It will be easier to research features.

**Comment by jaxxtheone** [comment]
Never mind, I SOLVED by myself. Thanks!

**Reply by antishov** [reply]
Hello,
Could you please share your workaround?

**Reply by jaxxtheone** [reply]
Hi!

There's no workaround. I just read on Bellaswag Github for GuppyFlo at [https://github.com/ballaswag/guppyflo](url) under Features, item 4, stating "Mpjeg-streamer webcams over tailscale. Don't use ngrok for these streams, they'll use all your free ngrok bandwidth".

**Reply by antishov** [reply]
Thanks for reply!
